FreydCategoriesForCAP/gap/CategoryOfRows.gd

```diff
@@ -76,7 +76,7 @@ DeclareOperation( RandomObjectByList, [ IsCategoryOfRows, IsList ] );
 DeclareOperation( RandomObjectByInteger, [ IsCategoryOfRows, IsInt ] );
 
 #! @Description
-#! The arguments are two objects <A>S</A>, <A>R</A> and a non-empty list <A>L</A> of non-negative integers.
+#! The arguments are two objects <A>S</A>, <A>R</A> and a list <A>L</A> of integers.
```

Comment:
Two things:
1. What about allowing any homalg ring element while your at it?
2. The documentation of the other `RandomMorphism*ByList` should also be adjusted.

Reply:
Addendum to 1.: Respectively any element that can be multiplied to homalg matrices of the given ring (which is not a condition we can check).

Reply:
Okay, why not!
The code is now relaxed to allow lists of integers and ring elements. I have also improved the error-messages in the derived-methods.